exist-core/src/main/java/org/exist/xquery/functions/fn/JSON.java

```diff
@@ -68,6 +88,7 @@
     public final static String OPTION_DUPLICATES_USE_LAST = "use-last";
     public final static String OPTION_LIBERAL = "liberal";
     public final static String OPTION_UNESCAPE = "unescape";
+    public static final QName KEY = new QName("key",null);
```

Comment:
A minor, stylistic suggestion: perhaps it's best to follow the order used on previous lines - `final static` instead of `static final`.

Reply:
The recommended formulation is `public static final`. Regardless, it should be consistent throughout ;-)